### PR TITLE
Add all GHA runs to dashboard

### PR DESCRIPTION
**Pull request by damccorm**
http://metrics.beam.apache.org/d/CTYdoxP4z/ga-pos…

Also exclude cancelled runs since many runs will fall into that category. Did limited testing since the dashboard is currently useless anyways, and prod testing is probably easier. May require a few iterations here.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by damccorm** [comment]
R: @Abacn

File: .test-infra/metrics/sync/github/sync_workflows.py [hunk 1]
```diff
@@ -134,13 +134,11 @@ def fetchWorkflowData():
...
-                    path =(result.group(1)) + ".yml"
-                    workflowObject = Workflow(item['id'],item['name'],path)
-                    WORKFLOWS_OBJECT_LIST.append(workflowObject)
+                path = item['path']
+                result = re.search('/(.*).yml', path)
+                path = (result.group(1)) + ".yml"
+                workflowObject = Workflow(item['id'],item['name'],path)
+                WORKFLOWS_OBJECT_LIST.append(workflowObject)
             url = "https://api.github.com/repos/apache/beam/actions/workflows/"
             queryOptions = { 'branch' : 'master', 'per_page' : GH_WORKFLOWS_NUMBER_EXECUTIONS,
```
Comment (Abacn) [inline .test-infra/metrics/sync/github/sync_workflows.py:143]: our phrase trigger implementation also runs the workflow firstly on master - then checkout the PR inside the workflow, and might be reported as master. Is it able to make the query to event:schedule only?

Comment (damccorm) [reply .test-infra/metrics/sync/github/sync_workflows.py:143]: My concern is that some workflows don't have scheduled runs (e.g. the ones around release automation)

Comment (Abacn) [reply .test-infra/metrics/sync/github/sync_workflows.py:143]: The if the trigger phrase does not match the workflow, it is skipped; if it matches the workflow runs to end, so I expect it won't get cancelled

I would understand the workflows does not have scheduled run not in the scope of "PostCommit" - and they are handful of them and we can either use different filter (if workflow name does not match PreCommit or PostCommit) or do not include them in postcommit dashboard

**Review by Abacn — APPROVED** [review]
Nevertheless this change fills a gap of the visibility for the GHA postcommits result so we can get it in and see how it looks like

File: .test-infra/metrics/sync/github/sync_workflows.py (hunk 1 above)
Comment (damccorm) [reply .test-infra/metrics/sync/github/sync_workflows.py:143]: Ah got it. Agreed on getting this in, but I can follow up to either filter to scheduled runs or just filter out runs with a comment trigger (I wish I could just do an excludes filter, but the API doesn't support it)

**Pull request merged**